joss_paper/paper.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'alchemlyb: the simple alchemistry library'
+tags:
+  - Python
+  - alchemistry
+  - molecular dynamics
+  - free energy
+authors:
+  - name: Zhiyi Wu
+    orcid: 0000-0002-7615-7851
+    equal-contrib: true
+    affiliation: 1
+  - name: David L. Dotson
+    orcid: 0000-0001-5879-2942
+    equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
+    affiliation: 2
```

Comment:
@dotsdl you did most of the work while at ASU and while on the NIH grant so it would be appropriate to also list the ASU affiliation and then mark Datryllic LLC as "present address" (there's probably a way to do it).

Would you agree?